targets/nintendoswitch.s

```diff
@@ -6,20 +6,19 @@
 _start:
     b start
     .word _mod_header - _start
-    .word 0
-    .word 0
+    .ascii "HOMEBREW"
 
     .ascii "NRO0"              // magic
     .word 0                    // version (always 0)
     .word __bss_start - _start // total NRO file size
     .word 0                    // flags (unused)
 
     // segment headers
-    .word __text_start
+    .word 0 // __text_start
     .word __text_size
-    .word __rodata_start
+    .word 0 //__rodata_start
     .word __rodata_size
-    .word __data_start
+    .word 0 //__data_start
```

Comment:
Thats necessary for NRO environment (which isn't the case of the emulator that is a NSO environment). For now, I'm not sure how to fill that, since linker complains that can't put a relocatable symbol there :(

Reply:
So with this PR can a TinyGo program still run correctly in the emulator?

Reply:
Yes, it does work fine. The screenshot is with this PR running on Emulator ^^